store/tikv/txn.go

```diff
@@ -49,6 +49,11 @@ var (
 	tikvTxnCmdHistogramWithLockKeys = metrics.TiKVTxnCmdHistogram.WithLabelValues(metrics.LblLockKeys)
 )
 
+const (
+	SlowAcquirePessimisticLock = 50 * time.Millisecond
+	SlowReleasePessimisticLock = 20 * time.Millisecond
```

Comment:
How about making them configurable? So, if the threshold is found to be not appropriate in production, we can adjust it.

Reply:
Done.